src/views/EdgeApplicationsDeviceGroups/Drawer/index.vue

```diff
@@ -103,6 +103,49 @@
     showEditDrawer.value = false
   }
 
+  const handleTrackSuccessEdit = () => {
```

Comment:
Can we move all these JS methods to a .js file and add tests to it?

Reply:
We have already tested that they cover this method, it is part of an already built class.
Creating JS files to group these functions would mean creating a method to access a method of a class, it brings complexity and ends up not making much sense.